node/src/lib.rs

```diff
@@ -2,11 +2,13 @@
 // SPDX-License-Identifier: Apache-2.0
 use config::{Parameters, SharedCommittee, WorkerId};
 use consensus::{bullshark::Bullshark, dag::Dag, Consensus, SubscriberHandler};
-use crypto::traits::{KeyPair, Signer, VerifyingKey};
+use crypto::traits::{EncodeDecodeBase64, KeyPair, Signer, VerifyingKey};
```

Comment:
I'd love to see a rebase because of this an many other small errors (this import is unneeded).

Reply:
Actually it's used to name the dhat-heap profile filename so it has a unique filename for each node.